firmware/src/widgets/parqetwidget/ParqetWidget.h

```diff
@@ -43,15 +63,21 @@ class ParqetWidget : public Widget {
     unsigned long m_clockDelayPrev = 0;
 
     String m_modes[PARQET_MODE_COUNT] = {"today", "1w", "1m", "3m", "6m", "1y", "3y", "mtd", "ytd", "max"}; // Possible timeframes: today, 1w, 1m, 3m, 6m, 1y, 3y, mtd, ytd, max
-    int8_t m_curMode = 0;
+    int m_defaultMode = 0;
+    int m_curMode = 0;
 
     boolean m_showClock = true; // Show clock on first screen
     boolean m_showTotalScreen = true; // Show a total portfolio screen
     boolean m_showTotalValue = false; // Show your total portfolio value
     boolean m_showTotalChart = true; // Show performance chart for total (if we have more than 7 datapoints, ie. not for "today")
     String m_overrideTotalChartToday = "1w"; // Show this chart for "today" to have a chart there as well, set to empty string to disable
-    boolean m_showValues = false; // Show current price (false) or value in portfolio (true)
+    int m_showValues = 0; // Show current price (0) or value in portfolio (1)
```

Comment:
An enum would be more convenient for this, that saves having to comment what is what?

Reply:
Same here. I'm using a ComboBox config parameter and that expects an &int as value.